makefiles/at-spi2-atk.mk

```
@@ -0,0 +1,79 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS         += at-spi2-atk
+AT-SPI2-ATK_VERSION := 2.38.0
+DEB_AT-SPI2-ATK_V   ?= $(AT-SPI2-ATK_VERSION)
+
+at-spi2-atk-setup: setup
+	wget -q -nc -P$(BUILD_SOURCE) https://download-fallback.gnome.org/sources/at-spi2-atk/$(shell echo $(AT-SPI2-ATK_VERSION) | cut -f-2 -d.)/at-spi2-atk-$(AT-SPI2-ATK_VERSION).tar.xz
+	$(call EXTRACT_TAR,at-spi2-atk-$(AT-SPI2-ATK_VERSION).tar.xz,at-spi2-atk-$(AT-SPI2-ATK_VERSION),at-spi2-atk)
+	mkdir -p $(BUILD_WORK)/at-spi2-atk/build
+	echo -e "[host_machine]\n \
+	cpu_family = '$(shell echo $(GNU_HOST_TRIPLE) | cut -d- -f1)'\n \
+	cpu = '$(MEMO_ARCH)'\n \
+	endian = 'little'\n \
+	system = 'darwin'\n \
+	[properties]\n \
+	root = '$(BUILD_BASE)'\n \
+	needs_exe_wrapper = true\n \
+	[paths]\n \
+	prefix ='$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)'\n \
+	[binaries]\n \
+	c = '$(CC)'\n \
+	objc = '$(CC)'\n \
+	cpp = '$(CXX)'\n \
+	pkgconfig = '$(BUILD_TOOLS)/cross-pkg-config'\n" > $(BUILD_WORK)/at-spi2-atk/build/cross.txt
+
+ifneq ($(wildcard $(BUILD_WORK)/at-spi2-atk/.build_complete),)
+at-spi2-atk:
+	@echo "Using previously built at-spi2-atk."
+else
+at-spi2-atk: at-spi2-atk-setup atk at-spi2-atk dbus glib2.0
```

Remove the cyclic dependency
```suggestion
at-spi2-atk: at-spi2-atk-setup atk dbus glib2.0 libxtst
```